src/components/elements/off-canvas-menu.js

```diff
@@ -83,7 +83,7 @@ const StyledLink = styled((props) => <LocalizedLink {...props} />)`
 const OffCanvasMenuContainer = styled.div`
     display: flex;
     flex-direction: column;
-    padding: 4rem 1.6rem;
+    padding: 0 1.6rem 21rem;
```

Comment:
instead of this, can we do overflow scroll?

Reply:
It will show the same issue. Partner section won't be displayed Please refer to 
![simulator_screenshot_AD4E8AAB-666B-4ABA-A3C5-DCDAACA3E08D](https://user-images.githubusercontent.com/29834890/100569326-a2dbc980-3308-11eb-9b13-94cee000b80b.png)